tests/commands/json/test_map.py

```diff
@@ -22,39 +25,6 @@
 from . import assert_command
 
 
-async def test_getMapSubSet_customName() -> None:
```

Comment:
This test tests the custom map name, and we should keep it.

Reply:
true, to much revert/removed